eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/ClusteringWriter.java

```diff
@@ -271,6 +290,27 @@ private SubmittedVariantOperationEntity buildSubmittedOperation(SubmittedVariant
         return operation;
     }
 
+    /**
+     * From EVA-2071, do not cluster submitted variants into a multimap clustered variant.
```

Comment:
I would explicitly add to this comment (or elsewhere if more relevant) that the submitted variants are left unclustered at the moment. Potentially that this needs to be revisited later.